openwisp_monitoring/db/backends/influxdb/queries.py

```diff
@@ -2,37 +2,38 @@
     'uptime': {
         'influxdb': (
             "SELECT MEAN({field_name})*100 AS uptime FROM {key} WHERE "
-            "time >= '{time}' AND content_type = '{content_type}' AND "
+            "time >= '{time}' AND time < '{time}' + {timespan} AND content_type = '{content_type}' AND "
             "object_id = '{object_id}' GROUP BY time(1d)"
```

Comment:
~Selecting single date on home page (`https://demo.openwisp.io/admin/`) simply breaks:~

Wait, it's not just single date, the thing worked once and now the home page custom range is simply broken! :-O

![dev-01](https://user-images.githubusercontent.com/24985760/185695515-bd56719e-678f-4c29-bf4f-c462ebd6ad9f.png)

Reply:
The demo system is not updated with the latest commit ig. I have to make sure the build pass and then we can have that on the demo. :+1: